bentoml/_internal/models/model.py

```diff
@@ -172,7 +172,9 @@ def create(
         Returns:
             object: Model instance created in temporary filesystem
         """
-        tag = Tag(name).make_new_version()
```

Comment:
let's update the inline docstring above to say it also accepts "tag"?

should we rename all the param `name` in save_models, to `tag`? or should we just keep it a hidden feature this way so we don't encourage users to use it?

Reply:
I don't know that we want to expose this behavior. Maybe we should add a kwarg that we can warn people about?

Reply:
I agree we don't want to expose this behavior, I think the way it is should be good